site/content/_index.md

```diff
@@ -1,52 +1,69 @@
-# Code Review During Development
+# Code Review During Research
 
-From small data analysis scripts to large simulation software, **quality
-software is essential to quality research**. Code readability,
-maintainability and extensibility are key to research outputs that
-can be reproduced and built upon.
+What is *code review*?
+If you're a researcher, why should you care about it?
+For many researchers, the term may not be familiar, 
+even for scientists that routinely write code.
+For software engineers in industry and academia, 
+the term "code review" brings to mind an established series of 
+good practices.
+
+At its core, code review consists of  
+**someone else reading code you wrote,
+and providing feedback**.
+When it works, this process makes code
+**easier to read, easier to maintain,
+and easier to extend**.
+
+This set of practices that software engineers call "code review"
+applies when software is mature and the scope of the project is clearly defined.
+Unfortunately, when writing code for research,
+the software is not mature, and the scope is not well understood.
+If either of those were true, it wouldn't be research.
+It is still unclear how well code review  
+translates to what is arguably the most common research setting:
+a single scientist, writing analysis code, 
+or helping to maintain a shared set of homegrown tools used by just one lab.
+Some work has been done to answer the question of how to translate code review to this setting  
+(please see [references and related work](./refs-related.md)).
+Although there are good ideas on how to adapt code review for researchers, 
+many of the efforts have been disparate.
```

Comment:
Even if this paragraph is true, this could be quite discouraging? As a researcher, why am I even reading this?

I'd rather be straight to the point here:
1. Code review is great and you should be interested in it (previous paragraph)
2. Although code review is comon software engineering practice it is still very rare in academia. This is for multiple reasons. For instance most guidance and good practices apply to team of develpoers working on single codebase with well defined scope and roles. Another one is the lack of incentive for researchers to learn/engage in code review.
3. We provide this guidance for researchers to get started with code reivew. Especially lone coders. 
4. Work on code review for research is still very disparate and this document should be regarded as opiniated guidelines instead of scientific truth. Nonetheless we base out work on a series of studies.
5. This is a living document people are free to contribute to.

Reply:
I agree with @tlestang  and I think he has made all the relevant points that I would. We should write the intro along the bullet points that he has outlined, with perhaps these amendments:

1. Code review is great and you should be interested in it (previous paragraph)
    - give the top headline reasons why (e.g. readability leading to maintainability, and more robust code)
1. Although code review is common software engineering practice it is still very rare in academia. (for a variety of reasons that I don't think we need to get into?)
    - We might want to mention that there are some efforts to do code review **at the time of publication**, but this is very different from the lightweight, incremental review that exists in industry
1. We provide **lightweight** guidance for researchers to get started with code review **during the research process**. Especially lone coders.
    - again, we need to emphasise _when_ the code review we are recommending should happen, and the fact that it doesn't need to be much of an additional burden that researchers can't fit into their already busy schedules.
1. Work on code review for research is still very disparate and this document should be regarded as opinionated guidelines instead of scientific truth. Nonetheless we base out work on a series of studies.
1. This is a living document people are free to contribute to. (and give details about how they can contribute)

Reply:
I think all 3 of us agree and it's a question of the order of presenting points